templates/users/new.rs.html

```diff
@@ -11,6 +11,7 @@ <h1>@i18n!(ctx.1, "Create an account")</h1>
         <form method="post">
             @input!(ctx.1, username (text), "Username", form, errors.clone(), "minlenght=\"1\"")
             @input!(ctx.1, email (text), "Email", form, errors.clone())
+            @input!(ctx.1, email (text), "Email", form, errors.clone(), "")
```

Comment:
I think we shouldn't ask for a custom domain when registering (I guess that's what you wanted to do here, even if it looks like you forgot to update what you copied)

Reply:
thanks for the catch, i changed this to:

```rust
            @input!(ctx.1, custom_domain (optional text), "custom_domain", form, errors.clone(), "")
```

and now i get this joyful error:

```
   --> src/template_utils.rs:127:30                                                                                                                                           
    |                                                                                                                                                                         
127 |                 val = escape(&$form.$name),                                                                                                                             
    |                              ^^^^^^ expected str, found enum `std::option::Option`                                                                                      
    |                                                                                                                                                                         
   ::: /home/meena/src/ap/Plume/target/debug/build/plume-7e8607967e0d30ae/out/templates/users/template_new.rs:21:1                                                            
    |                                                                                                                                                                         
21  | input!(ctx.1, custom_domain (optional text), "custom_domain", form, errors.clone(), "").to_html(out)?;                                                                  
    | --------------------------------------------------------------------------------------- in this macro invocation                                                        
    |                                                                                                                                                                         
    = note: expected type `&str`                                                                                                                                              
               found type `&std::option::Option<std::string::String>`                                                                                                         
                                                                                                                                                                              
error: aborting due to previous error                                                                                                                                         
                                                                                                                                                                              
For more information about this error, try `rustc --explain E0308`.                                                                                                           
error: Could not compile `plume`.                                                                                                                                             
```

Reply:
input! don't support `Option`, make `custom_domain` (in `NewUserForm`) not optional, and map it to None whenever it's empty in `create(...)`